src/Schema/TableDiff.php

```diff
@@ -189,9 +179,50 @@ public function __construct(
             );
         }
 
+        if (func_num_args() > 12 || (count($renamedIndexes) > 0 && current($renamedIndexes) instanceof Column)) {
+            Deprecation::trigger(
+                'doctrine/dbal',
+                'https://github.com/doctrine/dbal/pull/6080',
+                'Passing $renamedColumns to %s is deprecated and will be removed in the next major.',
```

Comment:
"Passing" is not something that can be removed. Either say that it will no longer be possible, or that the argument will be removed.